### PR TITLE
Correct link to Jupyter Code Executor in code-executors.ipynb

### LINKED ISSUE #2588: [Issue]: Incorrect link to Jupyter Code Executor
### Describe the issue

When clicking on the Jupyer Code Executor in the "Code Executors" page, you get 404

### Steps to reproduce

In [Code Executors](https://microsoft.github.io/autogen/docs/tutorial/code-executors/) click on the Jupyter Code Executor link

Expected: View the Jupyter Code Executor documentation
Actual: 404

### Screenshots and logs

image

image

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by erezak**
The link to the Jupyter code executor was referencing the wrong path.
Current: https://microsoft.github.io/autogen/docs/tutorial/topics/code-execution/jupyter-code-executor
Fixed: https://microsoft.github.io/autogen/docs/topics/code-execution/jupyter-code-executor

## Reviewer
@ekzhu 

## Why are these changes needed?

The link to the Jupyter Code Executor is broken

## Related issue number
Closes #2588 

## Checks

- [X] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [X] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [X] I've made sure all auto checks have passed.

### CODE REVIEW

File: website/docs/tutorial/code-executors.ipynb [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "\n",
     "In this chapter, we will focus on the command line code executors.\n",
     "For the Jupyter code executor, please refer to the topic page for \n",
-    "[Jupyter Code Executor](../topics/code-execution/jupyter-code-executor)."
+    "[Jupyter Code Executor](../../topics/code-execution/jupyter-code-executor)."
```
Comment (ekzhu) [inline website/docs/tutorial/code-executors.ipynb:44]:
```suggestion
    "[Jupyter Code Executor](/docs/topics/code-execution/jupyter-code-executor)."
```

File: website/docs/tutorial/code-executors.ipynb [hunk 2]
```diff
@@ -674,7 +674,7 @@
     "Contrast to the command line code executor, the Jupyter code executor\n",
     "runs all code blocks in the same Jupyter kernel, which keeps the state\n",
     "in memory between executions.\n",
-    "See the topic page for [Jupyter Code Executor](../topics/code-execution/jupyter-code-executor).\n",
+    "See the topic page for [Jupyter Code Executor](../../topics/code-execution/jupyter-code-executor).\n",
```
Comment (ekzhu) [inline website/docs/tutorial/code-executors.ipynb:677]:
```suggestion
    "See the topic page for [Jupyter Code Executor](/docs/topics/code-execution/jupyter-code-executor).\n",
```

**Review by ekzhu — CHANGES_REQUESTED** [review]
Use absolute path

**Review by Hk669 — APPROVED** [review]
LGTM.

**Pull request merged**